<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>1046</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skyrim Package Manager</string>
  </property>
  <widget class="QWidget" name="MainWindowWidget">
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="0" column="0">
     <widget class="QGroupBox" name="MainWindowGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QTabWidget" name="MainTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="PackagesAndSourcesTab">
          <attribute name="title">
           <string>Packages And Sources</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QGroupBox" name="PackageBrowserGroupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QSplitter" name="PackagesAndSourcesHSplitter">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <widget class="QSplitter" name="splitter">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QGroupBox" name="PackagesGroupBox">
                  <property name="title">
                   <string>Packages</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QListWidget" name="AliasesList"/>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QGroupBox" name="TarballsGroupBox">
                  <property name="title">
                   <string>Tarballs</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QListWidget" name="BlobsList"/>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QGroupBox" name="SourcesGroupBox">
                  <property name="title">
                   <string>Sources</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <widget class="QListWidget" name="SourcesList"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
                <widget class="QGroupBox" name="SourcesAreaGroupBox_3">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <widget class="QTabWidget" name="SourcesTabs">
                    <property name="currentIndex">
                     <number>0</number>
                    </property>
                    <widget class="QWidget" name="NexusSourcesTab">
                     <attribute name="title">
                      <string>NexusMods</string>
                     </attribute>
                     <layout class="QGridLayout" name="gridLayout_10">
                      <item row="0" column="0">
                       <widget class="QGroupBox" name="NexusInfo">
                        <property name="title">
                         <string/>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                         <item>
                          <widget class="QLineEdit" name="NexusUrl"/>
                         </item>
                         <item>
                          <widget class="QLabel" name="NexusTitle">
                           <property name="font">
                            <font>
                             <family>MS Sans Serif</family>
                             <pointsize>12</pointsize>
                             <weight>75</weight>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                           <property name="textFormat">
                            <enum>Qt::RichText</enum>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignCenter</set>
                           </property>
                           <property name="wordWrap">
                            <bool>true</bool>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLabel" name="NexusSummary">
                           <property name="font">
                            <font>
                             <family>MS Sans Serif</family>
                             <italic>true</italic>
                            </font>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                           <property name="textFormat">
                            <enum>Qt::RichText</enum>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignCenter</set>
                           </property>
                           <property name="wordWrap">
                            <bool>true</bool>
                           </property>
                           <property name="margin">
                            <number>6</number>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QGroupBox" name="groupBox">
                           <property name="title">
                            <string/>
                           </property>
                           <layout class="QGridLayout" name="gridLayout_4">
                            <item row="0" column="0">
                             <widget class="QTextEdit" name="NexusDescription">
                              <property name="textInteractionFlags">
                               <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                              </property>
                             </widget>
                            </item>
                            <item row="1" column="0">
                             <widget class="QTableWidget" name="NexusAvailableFiles">
                              <property name="editTriggers">
                               <set>QAbstractItemView::NoEditTriggers</set>
                              </property>
                              <property name="selectionBehavior">
                               <enum>QAbstractItemView::SelectRows</enum>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="GenericSourcesTab">
                     <attribute name="title">
                      <string>Generic</string>
                     </attribute>
                     <layout class="QGridLayout" name="gridLayout_13">
                      <item row="0" column="0">
                       <widget class="QLabel" name="label">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Packages_Folder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Packages_Folder">
   <property name="text">
    <string>Open Packages Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
